service/src/config/capabilityStatementResources.json

```diff
@@ -141,7 +141,7 @@
             }
           ],
           "name": "package",
```

Comment:
Do we want to change the name here? I am not sure so I thought I'd ask

Reply:
Yes, for consistency I think it's a good idea to update this!

Reply:
Honestly maybe because we are keeping the `package` name in `serverConfig.ts` should we change this back? I am sorry about all this hahah